packages/shared/routes/login/views/EnterPinView.svelte

```diff
@@ -106,23 +110,15 @@
                 .then((verified) => {
                     if (verified === true) {
                         return Platform.getMachineId().then(() =>
-                            getStorageDirectoryOfProfile(profile.id).then((path) => {
-                                initialiseProfileManager(path, COIN_TYPE[profile.networkProtocol], clientOptions, {
-                                    Stronghold: { snapshotPath: `${path}/wallet.stronghold` },
-                                })
-                                // TODO: set storage password with profile manager api
-                                // api.setStoragePassword(pinCode, {
-                                //     onSuccess() {
-                                //         dispatch('next')
-                                //     },
-                                //     onError(err) {
-                                //         isBusy = false
-                                //         showAppNotification({
-                                //             type: 'error',
-                                //             message: locale(err.error),
-                                //         })
-                                //     },
-                                // })
+                            buildProfileManagerOptionsFromProfileData(profile).then((profileManagerOptions) => {
+                                const { storagePath, coinType, clientOptions, secretManager } = profileManagerOptions
```

Comment:
This is overriding the client options from above and I am unable to login to an existing profile

Reply:
If we move the client options fallback to default client options a bit later, it should be fine.

Reply:
I am still receiving the same error message on older profiles. We need to have a fallback so that users can log in to older beta profiles if the client options are wiped?

Reply:
This is error is also showing up when creating a new software profile

Reply:
> We need to have a fallback so that users can log in to older beta profiles if the client options are wiped?

Definitely - I didn't remove the logic to get default client options but rather moved it to fit the new code.

> This is error is also showing up when creating a new software profile

Any particular steps? Did you skip the stronghold backup? When exactly did the error show up? I'm not getting any errors either on login or creation of new profiles.

Reply:
Did you run `yarn` in root to update everything?